dali/test/python/test_pipeline.py

```diff
@@ -657,7 +657,7 @@ def __init__(self, batch_size, num_threads, device_id, num_gpus):
                                              num_threads,
                                              device_id)
             self.input = ops.CaffeReader(path = caffe_db_folder, shard_id = device_id, num_shards = num_gpus)
-            self.decode = ops.ImageDecoder(device = "mixed", output_type = types.RGB)
+            self.decode = ops.ImageDecoder(device = "mixed", output_type = types.RGB, hw_decoder_load = 0.0)  # Fixing HW load to 0 to be able to compare with ImageDecoderCrop (which uses CUDA decoder only)
```

Comment:
Just a minor comment: can those comments appear in separate lines to not make them so long?

Reply:
done